test/run_tests.pl

```diff
@@ -62,7 +62,9 @@ BEGIN
     @tests = map { abs2rel($_, rel2abs(curdir())); } @tests;
 
     my $harness = $TAP_Harness->new(\%tapargs);
-    $harness->runtests(sort @tests);
+    my $ret = $harness->runtests(sort @tests);
+
+    exit $ret->has_errors if (ref($ret) eq "TAP::Parser::Aggregator");
```

Comment:
> Note that Test::Harness::runtests doesn't return anything,

I spoke too soon. Actually adding exit $ret in absence of TAP::Harness makes `make test` report failure. For reference original rationale was "keep old way the way it originally was [i.e. without any exit]".